src/EnergyPlus/HVACManager.cc

```diff
@@ -243,8 +243,8 @@ namespace EnergyPlus::HVACManager {
 
         state.dataHeatBalFanSys->ZT = state.dataHeatBalFanSys->MAT;
         // save for use with thermal comfort control models (Fang, Pierce, and KSU)
-        state.dataHeatBalFanSys->ZTAVComf = state.dataHeatBalFanSys->ZTAV;
-        state.dataHeatBalFanSys->ZoneAirHumRatAvgComf = state.dataHeatBalFanSys->ZoneAirHumRatAvg;
+        // state.dataHeatBalFanSys->ZTAVComf = state.dataHeatBalFanSys->ZTAV;
+        // state.dataHeatBalFanSys->ZoneAirHumRatAvgComf = state.dataHeatBalFanSys->ZoneAirHumRatAvg;
```

Comment:
@jcyuan2020 These lines should be deleted instead of commented, but otherwise, I'm good with this.

Reply:
@mjwitte The redundant commented code lines are now removed.